multicluster/docs/getting-started.md

```diff
@@ -0,0 +1,383 @@
+# Antrea Mulit-Cluster Controller Installation
```

Comment:
I feel we should put the multicluster docs into the root /docs directory. @antoninbas ?

Reply:
I think 2 solutions would work for me:
- add everything under `/docs/multicluster` at the repository root
- keep things as they are but add a link to this documentation in a placeholder `/docs/multicluster.md` document

there is a typo in the heading: s/Mulit-Cluster/Multi-Cluster